session-1/assets/css/main.css

```diff
@@ -0,0 +1,35 @@
+.box {
+    position: relative;
+    width: 100%;
+    padding-right: 15px;
+    padding-left: 15px
+}
+
+.our-services {
+    margin-top: 75px;
+    padding-bottom: 30px;
+    padding: 0 60px;
+    min-height: 198px;
+    text-align: left;
+    border-radius: 10px;
+    background-color: #fff;
+    transition: all .4s ease-in-out;
+    box-shadow: 0 0 25px 0 rgba(20, 27, 202, .17)
+}
+
+.our-services:hover h1,
+.our-services:hover p,
+.our-services:hover ul li{
+    color: #fff
+}
+
+.box-card:hover {
+    box-shadow: 0 0 25px 0 rgba(20, 27, 201, .05);
+    cursor: pointer;
+    background-image: linear-gradient(-45deg, #27b88d 0%, #22dd73 100%)
+}
+@media screen and (max-width: 800px) {
```

Comment:
To aligned text in each viewport size, read this documentation [Text](https://getbootstrap.com/docs/5.1/utilities/text/#text-alignmention)